src/components/RestaurantListItem.js

```diff
@@ -0,0 +1,40 @@
+const RESTAURANT_CATEGORY_IMAGES = {
+  한식: `${import.meta.env.BASE_URL}assets/category-korean.png`,
+  중식: `${import.meta.env.BASE_URL}assets/category-chinese.png`,
+  일식: `${import.meta.env.BASE_URL}assets/category-japanese.png`,
+  양식: `${import.meta.env.BASE_URL}assets/category-western.png`,
+  아시안: `${import.meta.env.BASE_URL}assets/category-asian.png`,
+  기타: `${import.meta.env.BASE_URL}assets/category-etc.png`,
+};
```

Comment:
```suggestion
const BASE_URL = import.meta.env.BASE_URL ?? "/";
const ASSETS_PATH = `${BASE_URL}assets`;

const RESTAURANT_CATEGORY_IMAGES = {
  한식: `${ASSETS_PATH}/category-korean.png`,
  중식: `${ASSETS_PATH}/category-chinese.png`,
  일식: `${ASSETS_PATH}/category-japanese.png`,
  양식: `${ASSETS_PATH}/category-western.png`,
  아시안: `${ASSETS_PATH}/category-asian.png`,
  기타: `${ASSETS_PATH}/category-etc.png`,
};
```

이렇게 중복 표현을 조금 더 제거할 수 있을 것 같아요!